src/crepe/system/AnimatorSystem.h

```diff
@@ -26,6 +24,9 @@ class AnimatorSystem : public System {
 	 * looping).
 	 */
 	void update() override;
+
+private:
+	LoopTimer & timer = LoopTimer::get_instance();
```

Comment:
Je zou mij een heel groot plezier doen om `dt` door te geven als argument aan de `update()` methode (dus dan wordt het `update(double dt)` o.i.d.). Dan heb je deze reference niet nodig en kan je update() mooier worden.

Reply:
dit kan niet want update is override van system

Reply:
Je mag van mij best wel de basis klasse aanpassen, dat moet toch gebeuren.

Reply:
ja maar dan krijgt iedere system dt mee willen we dat? kan het ook vragen in de groep